packages/suite/src/components/wallet/PrivacyAccount/AnonymityLevelSlider.tsx

```diff
@@ -39,9 +34,8 @@ const Label = styled(P)<{ $offset?: number }>`
     opacity: 0.5;
 `;
 
-const WarningWrapper = styled.div`
-    margin-top: 24px;
-    margin-bottom: -14px;
+const WarningWrapper = styled(motion.div)`
+    /* margin-top: 24px; */
```

Comment:
you removed the extended click area for the slider, so it can be changed by clicking on the numbers no more 😞

Reply:
Ah, that's what it was for.... It was a counter-intuitive, perhaps I can make the numbers clickable ;)

Reply:
I believe it's nice to have some space above and below slider clickable so user is not forced to drag the ticker or point to thin line. I probably should have left a comment explaining why it was there...